library/Hasql/Private/Prelude.hs

```diff
@@ -51,7 +53,12 @@ import Contravariant.Extras as Exports
 -- either
 -------------------------
 import Control.Monad.Trans.Either as Exports
+-- prefer implementations in base (via base-prelude) over the ones in either
+#if MIN_VERSION_base(4,10,0)
+import Data.Either.Combinators as Exports hiding (fromLeft, fromRight)
```

Comment:
Right. I meant we should have just hidden these from "base". I.e.,

```
import BasePrelude as Exports hiding (fromLeft, fromRight)
```

But it's alright.